packages/client/src/components/ProductList/ProductList.component.js

```diff
@@ -8,8 +8,7 @@ export default function ProductList({
   isLoading,
   products,
   filteredProducts,
-  toggleFavorite,
-  favoriteProducts,
+  // toggleFavorite,
```

Comment:
Please don't comment out code, delete it if it isn't needed and if you change your mind, Git allows you to revert changes.